.github/workflows/integration.yml

```diff
@@ -271,6 +271,26 @@ jobs:
       - name: build proposals tests
         run: yarn build
         working-directory: a3p-integration
+      - name: zip up all core proposals
+        run: |
+          find . -type d -path "./a3p-integration/proposals/*/submission" | while read -r dir; do
+            parent_dir=$(basename "$(dirname "$dir")")
+
+            sanitized_parent_dir=${parent_dir//:/-}
+
+            zip_file="${sanitized_parent_dir}.zip"
+
+            zip -r "$zip_file" "$dir"
+
+            echo "Zipped $dir into $zip_file"
+          done
+      - name: archive core proposals
```

Comment:
consider doing this before `build proposals tests` because that's a very slow job and may if it fails won't it skip this step?

I think if you `yarn build:submissions` and then this step, it'll set upload the artifacts regardless. And maybe they'll be available before the test job is done

Reply:
I think we need `build proposals tests` step to generate these proposals: https://github.com/Agoric/agoric-sdk/actions/runs/10461300357/job/28969341078#step:8:194

Or you're suggesting we do `yarn build:submissions` separately so we can archive core proposals sooner? I'm not sure if we gain much from it, the purpose of this change is for CI to generate core proposals instead of generating them manually, which is what we're doing currently in the vaults core-eval runbook. If the tests failed, we probably shouldn't use the corresponding core proposals.

Reply:
yeah, separately. agree with your conclusion.